tracking_manager/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Akretion (https://www.akretion.com).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Tracking Manager",
+    "summary": """This module tracks all fields of a model,
+                including one2many and many2many ones.""",
+    "version": "17.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["Kev-Roche", "sebastienbeau"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "base",
+        "mail",
+    ],
```

Comment:
```suggestion
    "depends": ["mail"],
```
no need to set base as a dependency, `mail` already has it included